src/src/lib.rs

```diff
@@ -426,6 +426,16 @@ pub extern "C" fn roc_fx_cwd() -> RocList<u8> {
     }
 }
 
+#[no_mangle]
+pub extern "C" fn roc_fx_posixTime() -> roc_std::U128 {
+    let start = SystemTime::now();
+    let since_epoch = start
+        .duration_since(UNIX_EPOCH)
+        .expect("time went backwards");
+
+    roc_std::U128::from(since_epoch.as_millis())
+}
```

Comment:
This seems fine, but something to consider for the future is that we could avoid the "time went backwards" panic scenario with something like this: (copy/pasting what chatGPT said; I have not vetted this code at all!)

```rust
#[cfg(unix)]
extern crate libc;

#[cfg(unix)]
use libc::{clock_gettime, timespec, CLOCK_REALTIME};
#[cfg(windows)]
extern crate winapi;
#[cfg(windows)]
use winapi::um::timeapi::{timeGetTime, TIMECAPS};

use std::mem::MaybeUninit;

fn ms_since_epoch() -> u64 {
    #[cfg(unix)]
    {
        let mut ts = MaybeUninit::<timespec>::uninit();
        let res = unsafe { clock_gettime(CLOCK_REALTIME, ts.as_mut_ptr()) };

        if res == 0 {
            let ts = unsafe { ts.assume_init() };
            return (ts.tv_sec as u64) * 1000 + (ts.tv_nsec as u64) / 1_000_000;
        } else {
            panic!("Failed to get the current time");
        }
    }
    #[cfg(windows)]
    {
        let ms = unsafe { timeGetTime() };
        ms as u64
    }
}
```

Reply:
I decided to leave this using the rust std library, and added a TODO for someone to update in a later PR. I'm not confident I know enough C to do this correctly.